### PR TITLE
doc: Add information how to ignore cache files in external dependencies

### PR DESCRIPTION
**Pull request by Czaki**
Add information about `.chezmoiignore` in "Include dotfiles from elsewhere" documentation section. 

Closes: #3267

### CODE REVIEW

File: assets/chezmoi.io/docs/user-guide/include-files-from-elsewhere.md [hunk 1]
```diff
@@ -73,6 +73,11 @@ When using Oh My Zsh, make sure you disable auto-updates by setting
 `~/.oh-my-zsh` directory to drift out of sync with chezmoi's source state. To
 update Oh My Zsh and its plugins, refresh the downloaded archives.
 
+!!! note
+    If your external dependency uses cache files inside directory is unpacked the 
+    chezmoi will report changed files when try to use `chezmoi apply`. 
+    To remove this please use [`.chezmoiignore`](/reference/special-files-and-directories/chezmoiignore/) file.
+
```
Comment (halostatue) [inline assets/chezmoi.io/docs/user-guide/include-files-from-elsewhere.md:80]: You have trailing whitespace on your lines, but I would reword this in any case:

```markdown
!!! note

    If your external dependency target directory can contain cache files that are
    added during normal use, chezmoi will report that files have changed on
    `chezmoi apply`. To avoid this, add the cache directory to your
    [`.chezmoiignore`](/reference/special-files-and-directories/chezmoiignore/)
    file.

    For example, Oh My Zsh may cache completions in `.oh-my-zsh/cache/completions/`,
    which should be added to your `.chezmoiignore` file.

**Review by halostatue — CHANGES_REQUESTED** [review]

**Comment by halostatue** [comment]
Thank your for the submission. Please reword your commit message to match what is in your PR title and add the line `Closes: #3267` to the bottom of your commit message so that the issue prompting this change is resolved when this is merged.

**Comment by Czaki** [comment]
updated

File: assets/chezmoi.io/docs/user-guide/include-files-from-elsewhere.md [hunk 2]
```diff
@@ -73,6 +73,16 @@ When using Oh My Zsh, make sure you disable auto-updates by setting
 `~/.oh-my-zsh` directory to drift out of sync with chezmoi's source state. To
 update Oh My Zsh and its plugins, refresh the downloaded archives.
 
+!!! note
+    If your external dependency target directory can contain cache files that are
```
Comment (halostatue) [inline assets/chezmoi.io/docs/user-guide/include-files-from-elsewhere.md:77]: There is supposed to be a blank line after `!!!note`.

**Review by halostatue — CHANGES_REQUESTED** [review]

**Review by twpayne — COMMENTED** [review]
Thank you for the contribution @Czaki and thank you for the review @halostatue!

I think all comments are now resolved, but I defer to @halostatue for the final check.

**Review by halostatue — APPROVED** [review]
